control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -479,8 +479,13 @@ func (r *Reconciler) reconcileInitialOffset(ctx context.Context, channel *messag
 }
 
 func (r *Reconciler) getSubscriberConfig(ctx context.Context, channel *messagingv1beta1.KafkaChannel, subscriber *v1.SubscriberSpec) (*contract.Egress, error) {
+	subscriberURI := subscriber.SubscriberURI.String()
+	if subscriberURI == "" {
+		return nil, fmt.Errorf("empty subscriber URI")
```

Comment:
It;s a bit different on what we fail for the trigger - perhaps make that a bit more consistent?

https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/control-plane/pkg/reconciler/trigger/trigger.go#L269-L272

Reply:
Let me know if the last commit addresses your comment.